docs/en-us/dev/user_doc/guide/task/conditions.md

```diff
@@ -17,20 +17,20 @@ Drag in the toolbar<img src="/img/conditions.png" width="20"/>The task node to t
 ## Parameter
 
 - Node name: The node name in a workflow definition is unique.
-- Run flag: Identifies whether this node can be scheduled normally, if it does not need to be executed, you can turn on the prohibition switch.
-- Descriptive information: describe the function of the node.
-- Task priority: When the number of worker threads is insufficient, they are executed in order from high to low, and when the priority is the same, they are executed according to the first-in first-out principle.
-- Worker grouping: Tasks are assigned to the machines of the worker group to execute. If Default is selected, a worker machine will be randomly selected for execution.
-- Number of failed retry attempts: The number of times the task failed to be resubmitted. It supports drop-down and hand-filling.
-- Failed retry interval: The time interval for resubmitting the task after a failed task. It supports drop-down and hand-filling.
-- Timeout alarm: Check the timeout alarm and timeout failure. When the task exceeds the "timeout period", an alarm email will be sent and the task execution will fail.
-- Downstream tasks: Supports two branches for now, success and failure
-  - Success: When the Conditions task runs successfully, run this downstream task
-  - Failure: When the Conditions task runs fails, run this downstream task
-- Upstream condition selection: one or more upstream tasks can be selected for conditions
-  - Add the upstream dependency: Use the first parameter to choose task name, and the second parameter for status of the upsteam task.
-  - Upstream task relationship: we use `and` and `or` operators to handle complex relationship of upstream when multiple upstream tasks for Conditions task
+- Run flag: Identifies whether this node schedules normally, if it does not need to execute, select the `prohibition execution`.
+- Descriptive information: Describe the function of the node.
+- Task priority: When the number of worker threads is insufficient, execute in the order of priority from high to low, and tasks with the same priority will execute in a first-in first-out order.
+- Worker grouping: Assign tasks to the machines of the worker group to execute. If `Default` is selected, randomly select a worker machine for execution.
+- Times of failed retry attempts: The number of times the task failed to resubmit. You can select from drop-down or fill-in a number.
+- Failed retry interval: The time interval for resubmitting the task after a failed task. You can select from drop-down or fill-in a number.
+- Timeout alarm: Check the timeout alarm and timeout failure. When the task runs exceed the "timeout", an alarm email will send and the task execution will fail.
+- Downstream tasks selection: supports two branches: success and failure
```

Comment:
I found we use two `:` in one line, it is ok or a mistake?

Reply:
done changes